runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java

```diff
@@ -102,6 +102,7 @@
 	public static final int MTA_TELEKINETIC_GROUP_ID = 198;
 	public static final int CORP_DAMAGE = 13;
 	public static final int DESTROY_ITEM_GROUP_ID = 584;
+	public static final int XERICS_TALISMAN_MENU_GROUP_ID = 187;
```

Comment:
Is this interface only used for the xeric's talisman? I would think it is the same one as the spirit tree one.

Reply:
Ah you're correct, it is used for the spirit tree as well. What is a more appropriate generic name for that group ID?

Reply:
Along those same lines, I wonder if the wilderness obelisk menu (after Hard wildy diary) uses the same widget. Maybe `SCROLL_TELEPORT_MENU_GROUP_ID`?